app/validators/task_done_validator.rb

```diff
@@ -0,0 +1,11 @@
+class TaskDoneValidator < ActiveModel::EachValidator
```

Comment:
Why do you need this Validator?
Is it possible to save any data type to a boolean column in db? Seem like ActiveRecord handle this case.

Reply:
If we send any string or number it will be converted to a `true` (empty string and zero will be `false`).